package.json

```diff
@@ -44,5 +44,5 @@
   },
   "pre-commit": [
     "lint"
-  ]
+  ],
```

Comment:
let's remove this comma since technically JSON [can't have trailing commas](https://stackoverflow.com/questions/201782/can-you-use-a-trailing-comma-in-a-json-object?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa)

Reply:
shoot just merged but will fix and merge again! thanks for catching it :)